sdk/tests/js/webgl-test-utils.js

```diff
@@ -33,6 +33,10 @@ var loggingOff = function() {
   error = function() {};
 };
 
+const ENUM_NAME_REGEX = RegExp('[A-Z][A-Z0-9_]*');
+const ENUM_NAME_BY_VALUE = {};
```

Comment:
Hm, should this also be a Map? (Looks correct to me either way, though.)

Reply:
It can be, but it doesn't need to be.
I used a Map for the prototype so I didn't need to rely on its stringification.

Reply:
👍